src/main/resources/assets/createqol/lang/ru_ru.json

```
@@ -1,57 +1,268 @@
+
+  "createqol.statue.rotation.cape": "Поворот",
+  "createqol.statue.rotation.hat": "Положение головы",
+  "createqol.statue.rotation.left_sleeve": "Положение левой руки",
+  "createqol.statue.rotation.right_sleeve": "Положение правой руки",
+  "createqol.statue.rotation.left_pants_leg": "Положение левой ноги",
+  "createqol.statue.rotation.right_pants_leg": "Положение правой ноги",
+  "createqol.statue.hatTab": "Положение головы",
+  "createqol.statue.left_sleeveTab": "Положение рук",
+  "createqol.statue.left_pants_legTab": "Положение ног",
+
+  "createqol.statue.hideBackground": "Нажмите, чтобы",
+  "createqol.statue.hideBackground1": "скрыть интерфейс",
+
+  "createqol.statue.rotations.default": "Стоит",
+  "createqol.statue.rotations.attention": "Стойка",
+  "createqol.statue.rotations.walking": "Идёт",
+  "createqol.statue.rotations.running": "Бежит",
+  "createqol.statue.rotations.pointing": "Указывает",
+  "createqol.statue.rotations.blocking": "Блокирует",
+  "createqol.statue.rotations.lungening": "Выпад",
+  "createqol.statue.rotations.winning": "Демонстрирует",
+  "createqol.statue.rotations.sitting": "Сидит",
+  "createqol.statue.rotations.arabesque": "Арабеск",
+  "createqol.statue.rotations.cupid": "Купидон",
+  "createqol.statue.rotations.confident": "Решается",
+  "createqol.statue.rotations.salute": "Честь",
+  "createqol.statue.rotations.death": "Мертвец",
+  "createqol.statue.rotations.facepalm": "Фейспалм",
+  "createqol.statue.rotations.lazing": "Ленится",
+  "createqol.statue.rotations.confused": "Думает",
+  "createqol.statue.rotations.formal": "Серьёзен",
+  "createqol.statue.rotations.sad": "Грустит",
+  "createqol.statue.rotations.joyous": "Радуется",
+  "createqol.statue.rotations.stargazing": "Любуется",
+  "createqol.statue.rotations.block": "Защищается",
+  "createqol.statue.rotations.item": "Показывает",
+  "createqol.statue.rotations.t_pose": "T-поза",
+  "createqol.statue.rotations.a_pose": "A-поза",
+
+  "createqol.statue.inventoryTab": "Снаряжение",
+  "createqol.statue.rotationPresetTab": "Позы",
+  "createqol.statue.skinTab": "Настройка скина",
+  "createqol.statue.transformTab": "Положение манекена",
+  "createqol.statue.position": "Позиция",
+  "createqol.statue.scale": "Размер",
+  "createqol.statue.animationTab": "Анимация",
+  "createqol.statue.publishedAnimationTab": "Опубликованные анимации",
+
+  "createqol.statue.mainTab.desc": "Цветовое форматирование Minecraft можно активировать символом '&'",
+  "createqol.statue.skinTab.desc": "А вы знали, что скины Minecraft кодируются в Base64?",
+  "createqol.statue.transformTab.desc": "Изменение размера манекена не затрагивает его хитбокс",
+  "createqol.statue.inventoryTab.desc": "Предметы снаряжения манекена являются лишь проекциями, а потому не выпадают при его уничтожении",
+  "createqol.statue.rotationPresetTab.desc": "Готовые позы были взяты из датапака 'Armor Statues', доступного на https://vanillatweaks.net",
```

"Набор данных" в ванилле всё же чаще встречается мне кажется
```suggestion
  "createqol.statue.rotationPresetTab.desc": "Готовые позы были взяты из набора данных 'Armor Statues', доступного на https://vanillatweaks.net",
```

---

Ла-а-адно.